modules/theme/src/themes/dark/assets/images/icons/code-fork.svg

```diff
@@ -1,5 +1,5 @@
 <!--
- * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Unless the file is removed, AFAIK the initial year should not be changed.

Reply:
The previous icon was removed. It replaced with a new icon.

Reply:
Fixed